src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -252,6 +246,11 @@ typedef struct {
 } MPIDI_OFI_atomic_valid_t;
 
 typedef struct {
+    struct fid_domain *domain;
+    struct fid_av *av;
+    struct fid_ep *ep;
```

Comment:
Can you add some comment here on how `ep` is used? I think there are a relatively complicated cases, right? For multi-domain mode this would point to actual endpoint made out of each domain, whereas for scalable ep mode this points to scalable endpoint if ctx[0], invalid (NULL) otherwise. Correct?

Reply:
Will do.